lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2414,7 +2414,7 @@ def deploy_vm(self):
                 if key == 'type' and nw['type'] == 'dhcp':
                     network_changes = True
                     break
-                if key not in ('device_type', 'mac', 'name', 'vlan', 'type', 'start_connected'):
+                if key not in ('device_type', 'mac', 'name', 'vlan', 'type', 'start_connected', 'dvswitch_name'):
```

Comment:
Could you please add a changelog entry [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments)? LGTM

Reply:
changelog entry added. thanks!

Reply:
Thanks for the patch. We should actually use a whitelist, not a blacklist.